src/components/PostCard.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  Box,
+  Card,
+  CardBody,
+  CardHeader,
+  Heading,
+  ListItem,
+  Stack,
+  StackDivider,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+import { BlogPostCategory } from "../types/blogPost";
+
+export interface PostFormProps {
+  title: string;
+  score: number;
+  categories: BlogPostCategory[];
+  author: string;
+}
+
+export function PostCard({ categories, score, title }: PostFormProps) {
+  return (
+    <ListItem>
```

Comment:
`ListItem` here indicates that `PostCard` is an element of some list. This is not okat, because maybe you would like to use this component outside list. You should remove `ListItem` from here and move it to `PostList.tsx`.

For example code like this (I'm using `Tab` instead of `List` in this example)

```
const MyCompoent = () => {
  return (
    <Tabs>
      <Tab>
        <SomeComponentOne />
      </Tab>

      <Tab>
        <SomeComponentTwo />
      </Tab>
    </Tabs>
  );
};

const SomeComponentOne = () => {
  return <h1>SomeComponentOne</h1>;
};

const SomeComponentTwo = () => {
  return <h1>SomeComponentTwo</h1>;
};
```
In much better than 

```
const MyCompoent = () => {
  return (
    <Tabs>
      <SomeComponentOne />
      <SomeComponentTwo />
    </Tabs>
  );
};

const SomeComponentOne = () => {
  return (
    <Tab>
      <h1>SomeComponentOne</h1>
    </Tab>
  );
};

const SomeComponentTwo = () => {
  return (
    <Tab>
      <h1>SomeComponentTwo</h1>
    </Tab>
  );
};
```

Because `SomeComponentOne` and `SomeComponentTwo` can be used outside tabs